src/locales/gb.json

```
@@ -81,13 +56,100 @@
+	"settingsFailedToDeleteMonitors": "Failed to delete all monitors",
+	"starPromptTitle": "Star Checkmate",
+	"starPromptDescription": "See the latest releases and help grow the community on GitHub",
+	"https": "HTTPS",
+	"http": "HTTP",
+	"monitor": "monitor",
+	"aboutus": "About Us",
+	"signUP": "Sign Up",
+	"now": "Now",
+	"createYour": "Create your",
+	"createMonitor": "Create monitor",
+	"actions": "Actions",
```

Biome (1.9.4)

[error] The key actions was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)